apps/portal-api/src/modules/services/deployments/deployments.domain.ts

```diff
@@ -148,6 +148,14 @@ export const DeploymentRequestDomain = {
       .first();
   },
 
+  loadExpiredTrials: async (): Promise<DeploymentRequest[]> => {
```

Comment:
```suggestion
  loadTrialsToExpire: async (): Promise<DeploymentRequest[]> => {
```

Because the trials are not already expired since the hub status is still Active. What do you think ?